### DISCUSSION TITLE
What is the memory optimization limit for webpack in the future?

### DISCUSSION THREAD

**Discussion by i11010520** [webpack/webpack#18896] [General]
Is there some roadmap and progress for optimization?
Thanks guys.

**Comment by alexander-akait** [comment]
Try - https://github.com/webpack/webpack/blob/main/lib/config/defaults.js#L372C18-L372C33 (for this - https://github.com/webpack/webpack/blob/main/lib/config/defaults.js#L453), we want to enable it in near future
